mutations/gov/useSignMessage.ts

```diff
@@ -53,7 +53,7 @@ const useSignMessage = () => {
 		async (payload: SignaturePayload) => {
 			const version = '0.1.3';
 			let msg: any = {
-				address: walletAddress?.toLowerCase(),
+				address: ethers.utils.getAddress(walletAddress ?? ''),
```

Comment:
can you confirm `ethers.utils.getAddress('')` is not going to return an error?

Reply:
@clementbalestrat good pick up